x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/helpers.tsx

```diff
@@ -490,20 +490,29 @@ export const buildRuleTypeDescription = (label: string, ruleType: Type): ListIte
   }
 };
 
-export const buildThresholdDescription = (label: string, threshold: Threshold): ListItems[] => [
-  {
-    title: label,
-    description: (
-      <>
-        {isEmpty(threshold.field[0])
-          ? `${i18n.THRESHOLD_RESULTS_ALL} >= ${threshold.value}`
-          : `${i18n.THRESHOLD_RESULTS_AGGREGATED_BY} ${
-              Array.isArray(threshold.field) ? threshold.field.join(',') : threshold.field
-            } >= ${threshold.value}`}
-      </>
-    ),
-  },
-];
+export const buildThresholdDescription = (
+  label: string,
+  threshold: FieldValueThreshold
+): ListItems[] => {
+  let thresholdDescription = isEmpty(threshold.field[0])
+    ? `${i18n.THRESHOLD_RESULTS_ALL} >= ${threshold.value}`
+    : `${i18n.THRESHOLD_RESULTS_AGGREGATED_BY} ${
+        Array.isArray(threshold.field) ? threshold.field.join(',') : threshold.field
+      } >= ${threshold.value}`;
+  if (
+    threshold.cardinality &&
+    threshold.cardinality.value &&
+    threshold.cardinality.field.length > 0
+  ) {
+    thresholdDescription = `${thresholdDescription} ${i18n.THRESHOLD_CARDINALITY} ${threshold.cardinality.field[0]} >= ${threshold.cardinality.value}`;
+  }
+  return [
+    {
+      title: label,
+      description: <>{thresholdDescription}</>,
```

Comment:
```suggestion
      description: thresholdDescription,
```

No need for fragment in this case.

Reply:
Actually it complains when I remove the fragment. That's why I left it there in the first place. I was a bit confused to see this. I need to investigate later why it is like that.

Reply:
Weird. It doesn't complain for me if I remove the fragment. Let's take a look together.

Reply:
Sure, let's sync